### PR TITLE
Add clarification notes about Astro v Software CLI commands

### LINKED ISSUE #3196: Audit CLI Command reference
### Issue Type

Missing content for an existing feature

### Links to Affected Docs

https://docs.astronomer.io/astro/cli/reference

### The Issue or Context

There are some new options and functionality that is limited to either Astro or Software and not available to both using the Astro CLI. 

### Fix or Proposed Change

Conduct an audit of all CLI commands and determine which are for software only, astro only, both, and the different options between the two. 

### Additional Notes

_No response_

### Required Reviewers

_No response_

### PR DESCRIPTION
**Pull request by lzdanski**
Resolves #3196 

Base branch for additional CLI audits

### CODE REVIEW

File: astro/cli/astro-organization-team-update.md [hunk 1]
```diff
@@ -4,9 +4,15 @@ title: "astro organization team update"
 id: astro-organization-team-update
 description: Update a Team in your Organization.
 hide_table_of_contents: true
-sidebar_custom_props: { icon: 'img/term-icon.png' } 
+sidebar_custom_props: { icon: 'img/term-icon.png' }
 ---
 
+:::info
+
+This command is only available for Deployments on Astro.
```
Comment (jwitz) [inline astro/cli/astro-organization-team-update.md:12]: This isn't necessarily a Deployment-related command. For all commands like this, might be worth it to ctrl-f and replace this line to "This command is available only for Astro."

**Review by jwitz — APPROVED** [review]
Thank you so much for addressing this @lzdanski , I'm sure it was monotonous but it's absolutely worth it! I just had one minor comment about the format of the info note, but otherwise this LGTM. Great call on also including a note for when the command applies to both Astro and Software

**Comment by lzdanski** [comment]
@jwitz since the adjustment changed notes on 41 files, do you mind giving it a spot-look to triple check I got it right?

**Review by jwitz — APPROVED** [review]
LGTM

**Comment by lzdanski** [comment]
Hey @jwitz could you give this one final look in preview? It's such a big change I need a double check

File: astro/cli/astro-workspace-update.md [hunk 2]
```diff
@@ -36,3 +51,35 @@ $ astro workspace update --name "My Deployment" --enforce-ci-cd OFF
...
+</TabItem>
+<TabItem value="software">
+
+## Usage
+
+```sh
+astro workspace update <workspace-id> <options>
+```
+
+You can find a Workspace's ID by running `astro workspace list`, or by opening your Workspace and going to **Workspace Settings** > **General** in the Cloud UI. If you do not provide a Workspace ID, the CLI prompts you to pick from a list of Workspaces that you belong to in your current Organization.
```
Comment (jwitz) [inline astro/cli/astro-workspace-update.md:64]:
```suggestion
You can find a Workspace's ID by running `astro workspace list`. If you do not provide a Workspace ID, the CLI prompts you to pick from a list of Workspaces that you belong to in your current Organization.
```

File: astro/cli/astro-logout.md [hunk 3]
```diff
@@ -4,9 +4,15 @@ title: 'astro logout'
...
+sidebar_custom_props: { icon: 'img/term-icon.png' }
 ---
 
+:::info
+
+The behavior and format of this command are the same for both Astro and Software.
+
+:::
+
 Log out of the Astro CLI. This command does not affect your current login session in the Cloud UI.
```
Comment (jwitz) [inline astro/cli/astro-logout.md:16]:
```suggestion
Log out of the Astro CLI. This command does not affect your current web browser login session in either Astro or Astronomer Software.
```

File: astro/cli/astro-workspace-delete.md [hunk 4]
```diff
@@ -4,18 +4,24 @@ title: "astro workspace delete"
...
+Delete an Astro Workspace.
 
 ## Usage
 
 ```sh
 astro workspace delete <workspace-id>
 ```
 
-You can find a Workspace's ID by running `astro workspace list`, or by opening your Workspace and going to **Workspace Settings** > **General** in the Cloud UI. If you don't provide a Workspace ID, the CLI prompts you to pick from a list of Workspaces that you belong to in your current Organization. 
+You can find a Workspace's ID by running `astro workspace list`, or by opening your Workspace and going to **Workspace Settings** > **General** in the Cloud UI. If you don't provide a Workspace ID, the CLI prompts you to pick from a list of Workspaces that you belong to in your current Organization.
```
Comment (jwitz) [inline astro/cli/astro-workspace-delete.md:24]:
```suggestion
You can find a Workspace's ID by running `astro workspace list`. Alternatively, in Astro, you can find a Workspace ID by opening the Workspace and going to **Workspace Settings** > **General** in the Cloud UI. If you don't provide a Workspace ID, the CLI prompts you to pick from a list of Workspaces that you belong to in your current Organization.
```

**Review by jwitz — APPROVED** [review]
Found a couple of places where we mention Astro stuff in a Software context, otherwise LGTM!

File: astro/cli/astro-deployment-airflow-variable-copy.md [hunk 5]
```diff
@@ -7,6 +7,12 @@ hide_table_of_contents: true
 sidebar_custom_props: { icon: 'img/term-icon.png' }
 ---
 
+:::info
+
+This command is only available on Astro.
```
Comment (paolaperaza) [inline astro/cli/astro-deployment-airflow-variable-copy.md:12]: Add clarifying note here that Software doesn't support this feature?

**Pull request merged**